### PR TITLE
grafana: add grpc resource group QPS panel

### PR DESCRIPTION
**Pull request by Connor1996**
Signed-off-by: Connor1996 <zbk602423539@gmail.com>

### What is changed and how it works?

Issue Number: Ref #13730 

What's Changed:

```commit-message
Add grpc resource group QPS panel and fix datasource
```
截屏2023-02-07 16 18 03

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

### Release note 

```release-note
None
```

### CODE REVIEW

File: metrics/grafana/tikv_details.json [hunk 1]
```diff
@@ -5770,7 +5763,112 @@
...
+          "points": false,
+          "renderer": "flot",
+          "seriesOverrides": [],
+          "spaceLength": 10,
+          "stack": false,
+          "steppedLine": false,
+          "targets": [
+            {
+              "exemplar": true,
+              "expr": "sum(rate(tikv_grpc_resource_group_total{k8s_cluster=\"$k8s_cluster\", tidb_cluster=\"$tidb_cluster\", instance=~\"$instance\"}[1m])) by (name)",
```
Comment (JmPotato) [inline metrics/grafana/tikv_details.json:5834]: Will `[30s]` interval be more accurate?

Comment (glorv) [reply metrics/grafana/tikv_details.json:5834]: +1

**Review by glorv — COMMENTED** [review]
LGTM

**Review by glorv — APPROVED** [review]

**Review by BornChanger — APPROVED** [review]

**Review by JmPotato — APPROVED** [review]

**Review by tabokie — APPROVED** [review]

**Pull request merged**